examples/react-native-example/App.tsx

```diff
@@ -71,16 +71,9 @@ export default function App() {
   }
 
   if (view && linkToken?.length) {
-    console.log(linkToken, 'linkToken');
-
     return (
       <LinkConnect
         linkToken={linkToken}
-        onIntegrationConnected={(payload: LinkPayload) => {
-          if (payload.accessToken) {
-            showIntegrationConnectedAlert(payload.accessToken);
-          }
-        }}
         accessTokens={[
```

Comment:
Property 'accessTokens' does not exist on type 'IntrinsicAttributes & LinkConfiguration'.ts(2322)

Reply:
This was still accessing the old version. I have updated the example code and readme to reflect the updated codebase